changelog/db.changelog.xml

```diff
@@ -14,4 +14,5 @@
   <include  file="changelog/db.changelog-1.7.xml"/>
   <include  file="changelog/db.changelog-1.8.xml"/>
   <include  file="changelog/db.changelog-1.9.xml"/>
+  <include  file="changelog/db.changelog-1.10.0.xml"/>
```

Comment:
any reason for 1.10.0 rather than 1.10 - just feels like its going to get confusing